ush/forecast_postdet.sh

```diff
@@ -327,6 +329,33 @@ EOF
     done
   fi
   #============================================================================
+  restart_interval=${restart_interval:-${FHMAX}}
+  # restart_interval = 0 implies write restart at the END of the forecast i.e. at FHMAX
+  # Convert restart interval into an explicit list for CMEPS/CICE/MOM6/WW3
+  # Note, this must be computed after determination IAU in forecast_det and fhrot.
+  if (( restart_interval == 0 )); then
+    if [[ "${DOIAU:-NO}" == "YES" ]]; then
+      FV3_RESTART_FH=$(( FHMAX + half_window ))
+    else
+      FV3_RESTART_FH=("${FHMAX}")
```

Comment:
Ah, I get it now.  I think that if `FV3_RESTART_FH` is a single hour (i.e. `DOIAU=NO`), it should be a string, not an array of length 1:

```suggestion
      FV3_RESTART_FH="${FHMAX}"
```

Reply:
Maybe I'm wrong here.  Let me think about this...

Reply:
Note this is very similar to how the CMEPS_RESTART_FH is calculated, which is how I knew what shell check disable to use.

Reply:
OK, my confusion came from this line in parsing_model_configure.sh:
https://github.com/NOAA-EMC/global-workflow/blob/464a1e2e412c5dd333f82b7423eb4c6b5da8fc62/ush/parsing_model_configure_FV3.sh#L27

This is the only location that `FV3_RESTART_FH` is referenced and it turns the variable (whether a string or an array) into a single string separated by spaces.  Now that I understand that, I'm OK with this change for now.  Though I think the array usage should be cleaned up in the `forecast_*` scripts and `parsing_model_configure.sh`.  I'll open a separate issue for that.

Reply:
Opened https://github.com/NOAA-EMC/global-workflow/issues/3978